src/proto/project.proto

```diff
@@ -51,6 +57,7 @@ message Project {
   uint64 created_at = 8;             // Time of project creation (UTC seconds epoch)
   uint64 updated_at = 9;             // Time of last project update (UTC seconds epoch)
   repeated ProjectAsset assets = 10; // The listing of all assets associated with the project
+  optional ProjectType type = 11; // The type of project
```

Comment:
Optional seems redundant.  With optional omitted, the enum will still default to the unspecified case.

Reply:
Yeah nice call. I probably did this because I didn't backfill the table with the default value. I suppose that doesn't matter though, ultimately, as the default value will get set for records without a value.

Reply:
Addressed in most recent change.